pages/api/_middleware.ts

```diff
@@ -19,7 +22,7 @@ export async function middleware(req: NextApiRequest) {
     info({method, ip, ua, ...page, pathname, search, body});
 
 
-  if(method !== 'GET' && !isInWhiteList){
+  if(method !== 'GET' && !isInWhiteList || serverRuntimeConfig?.e2e?.testnet !== true){
```

Comment:
Note: this condition will always be true in production and thus it won't respect the previous condition. What exactly is it that you want to achieve with this condition ?